### PR TITLE
Toast position fixed

### PR DESCRIPTION
**Pull request by gagandeepp**
Fixes: https://github.com/ToolJet/ToolJet/issues/4384

Screenshot 2022-10-13 at 7 13 50 PM

### CODE REVIEW

**Comment by gagandeepp** [comment]
@withshubh Please review this.

**Comment by gagandeepp** [comment]
@arpitnath  Please review the PR

File: frontend/src/ManageOrgUsers/ManageOrgUsers.jsx [hunk 1]
```diff
@@ -174,7 +174,7 @@ class ManageOrgUsersComponent extends React.Component {
 
   invitationLinkCopyHandler = () => {
     toast.success('Invitation URL copied', {
-      position: 'bottom-right',
+      position: 'top-right',
```
Comment (gsmithun4) [inline frontend/src/ManageOrgUsers/ManageOrgUsers.jsx:177]: Position should be `top-center`

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

**Review by gsmithun4 — APPROVED** [review]

**Comment by Oluwatobiiiiii** [comment]
The link is not working

**Comment by withshubh** [comment]
Hey @gagandeepp  
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Comment by withshubh** [comment]
Hey @gagandeepp - We haven't received your submission for the hacktoberfest swag. Please submit the form as soon as possible.

**Comment by withshubh** [comment]
Hey @gagandeepp - Please submit the form before the end of this week to claim your Hacktoberfest swag: [airtable.com/shr1qR6xhqsWO7LD7](https://airtable.com/shr1qR6xhqsWO7LD7)

**Pull request merged**